webpack/assets/javascripts/react_app/components/notifications/Notification.test.js

```diff
@@ -0,0 +1,55 @@
+jest.unmock('./Notification');
+
+import React from 'react';
+import { mount } from 'enzyme';
+import Notification from './Notification';
+
+/* eslint-disable camelcase */
+const notification = {
+  id: 1,
+  text: 'Job well done',
+  level: 'success',
+  created_at: '2016-12-13 16:52:47Z'
+};
+/* eslint-enable camelcase */
+
+function setup(notification) {
+  return mount(<Notification {...notification} />);
+}
+
+describe('Notification', () => {
+  beforeEach(() => {
+    global.__ = (text) => text;
```

Comment:
Brilliant solution @gailsteiger, but maybe we should make it globally available? (at least the `__` thing)